components/BottomNavigation/tests/unit/BottomNavigationTests.m

```diff
@@ -163,4 +163,19 @@ -(void)testTitleVisibility {
   XCTAssert(self.bottomNavBar.itemViews.lastObject.label.isHidden);
 }
 
+-(void)testViewForItem {
+  UITabBarItem *item1 = [[UITabBarItem alloc] initWithTitle:@"1" image:nil tag:0];
+  UITabBarItem *item2 = [[UITabBarItem alloc] initWithTitle:@"2" image:nil tag:0];
+  UITabBarItem *item3 = [[UITabBarItem alloc] initWithTitle:@"3" image:nil tag:0];
+  self.bottomNavBar.items = @[item1, item2, item3];
+
+  XCTAssertEqual([self.bottomNavBar viewForItem:item1], self.bottomNavBar.itemViews[0]);
```

Comment:
Let's change these asserts to not require the ordering of `itemViews`. Instead, let's just assert that the returned object is an element of the array and that for two items they are not the same view.  Something like:

```objc
UIView *viewForItem1 = [self.bottomNavBar viewForItem:item1];
UIView *viewForItem2 = [self.bottomNavBar viewForItem:item2];

// Then
XCTAssertNotEqual(viewForItem1, viewForItem2);
XCTAssertTrue([self.bottomNavBar.itemViews containsObject:viewForItem1], @"BottomNavBar.itemViews did not contain the view (%@) returned for UITabBarItem (%@)", viewForItem1, item1);
XCTAssertTrue([self.bottomNavBar.itemViews containsObject:viewForItem2], @"BottomNavBar.itemViews did not contain the view (%@) returned for UITabBarItem (%@)", viewForItem2, item2);

```

Reply:
Sounds good.